### DISCUSSION TITLE
Just the 100 millionth pioneer to be mobbed.

### DISCUSSION THREAD

**Discussion by clort81** [rms-support-letter/rms-support-letter.github.io#5790] [General]
This is part of a social dynamic that's been going on for millennia.

Described aptly in Ayn Rand's "Atlas Shrugged".  The cliffs notes version suffices, as Rand isn't concise or highly readable.

A lot of you may be beginning to wake up to the reality of the 'new normal' they're trying to impose.  It's high time to start reading up on those things that you used-to reject as fringe.

**Comment by clort81** [comment]
"To the young man, every situation is specific, it demands attention to how it is different than other situations.  To the old man, every situation is part of a general class of similar situations." - clort

**Comment by 6r1d** [comment]
Hello. Yeah, can't help thinking the same way. I had much higher opinion of Mozilla, EFF and other guys before all that craze happened.

**Comment by clort81** [comment]
It's amusing that people who feign offence at Stallman's remarks about the grey zones of sexual consent have no problem reversing the gender of a child without his or her informed, adult consent.

But hypocrisy, contradiction and insanity is part of the tactic.

**Comment by antiwoke** [comment]
@clort81 so fucking true. They are blinded by their disgusting ideology.
